docs/guidance/batch-inference-example.md

```diff
@@ -0,0 +1,18 @@
+# RayJob Batch Inference Example
```

Comment:
What's this?

Reply:
Sorry I accidentally added file to git, let me remove it